pacmanconf/src/pacmanconf.rs

```diff
@@ -309,7 +313,13 @@ impl Config {
                 "VerbosePkgLists" => self.verbose_pkg_lists = true,
                 "DisableDownloadTimeout" => self.disable_download_timeout = true,
                 "UseDelta" => self.use_delta = 0.7,
-                "DisableSandbox" => self.disable_sandbox = true,
+                "DisableSandbox" => {
+                    self.disable_sandbox = true;
+                    self.disable_sandbox_filesystem = true;
+                    self.disable_sandbox_syscalls = true;
+                }
+                "DisableSandboxFilesystem" => self.disable_sandbox_filesystem = true,
+                "DisableSandboxSyscalls" => self.disable_sandbox_syscalls = true,
```

Comment:
I don't think this should set the other fields as the struct just verbatim records what the config said. The fields are public and the user could set one of these fields without the other.